Sources/Models/OAFavoriteItem.mm

```diff
@@ -112,6 +112,9 @@ - (NSString *) getHumanString
 
 
 @implementation OAFavoriteItem
+{
+    NSString *_name;
```

Comment:
1. Introduce cache for other fields like Description, Address, etc...
2. Reset cache values at `set<Name>` methods